packages/data/json/buildingCode.json

```diff
@@ -0,0 +1,150 @@
+[
+  {
+    "section": "9.9.9.1",
+    "header": "Egress from Dwelling Units",
+    "content": {
+      "clauses": [
+        {
+          "description": "Except as provided in <tag>Sentences (2) and (3), every dwelling unit containing more than 1 storey shall have exits or egress doors located so that it shall not be necessary to travel up or down more than 1 storey to reach a level served by...",
+          "subsections": [
```

Comment:
Thoughts on the children of clauses would have "subclauses" instead of "subsections" ?

Reply:
naming is updated in BC ticket